swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md

```diff
@@ -74,21 +74,23 @@ The following table lists the Ballerina types supported by the MySQL data store
 |       enum       |      ENUM       |
 
  
-If you want to map a Ballerina type to a different SQL type or want to change the default length of an SQL type, you can change it in the `script.sql` file generated by the `persist generate` command before executing the script.
+This feature doesn't support mapping Ballerina types to different SQL types or changing the default length of an SQL type. If you want to map a Ballerina type to a different SQL type or want to change the default length of an SQL type, you can change it in the `script.sql` file generated by the `persist generate` command before executing the script. it may not work for all the types.
```

Comment:
```suggestion
This feature doesn't support mapping Ballerina types to different SQL types or changing the default length of an SQL type. If you want to map a Ballerina type to a different SQL type or want to change the default length of an SQL type, you can change it in the `script.sql` file generated by the `persist generate` command before executing the script. It may not work for all the types.
```